packages/lesswrong/server/notificationBatching.js

```diff
@@ -98,22 +99,36 @@ export const wrapAndSendEmail = async ({user, subject, body}) => {
 
 addGraphQLResolvers({
   Query: {
-    async EmailPreview(root, {notificationIds}, context) {
+    async EmailPreview(root, {notificationIds, postId}, context) {
```

Comment:
This would have been better as two different functions, rather than one function which takes two mutually exclusive arguments.

Reply:
I appreciate the attention and feedback.